### PR TITLE
ACDC blockscout

### PR DESCRIPTION
**Pull request by michellebrier**
### Description
POC blockscout for ACDC

### Usage
NOTE: do not deploy on a normal DN. This uses too much CPU when first indexing ACDC and crashes the node. It runs on its own dedicated node for now.
1. Stop and rm all containers running on the node except chain and caddy.
2. Generate a secret key base string to protect production assets:
```
SECRET_KEY_BASE=$(openssl rand -hex 32) && echo "SECRET_KEY_BASE='$SECRET_KEY_BASE'" > audius-docker-compose/discovery-provider/blockscout.env
```
3. Change `audius-docker-compose/discovery-provider/Caddyfile` to this:
```
{$audius_discprov_url}

{$CADDY_TLS}

encode zstd gzip

@blockscout_proxy {
    path / /_next* /node-api* /apps* /account* /accounts* /static* /auth/profile* /auth/unverified-email* /txs* /tx* /blocks* /block* /login* /address* /stats* /search-results* /token* /tokens* /visualize* /api-docs* /csv-export* /verified-contracts* /graphiql* /withdrawals* /l2-output-roots* /l2-txn-batches* /l2-withdrawals* /l2-deposits*
}

route @blockscout_proxy {
    reverse_proxy acdc-blockscout-frontend:3000
}

reverse_proxy acdc-blockscout-backend:4000
```
4. Register blockscout as a plugin:
```
audius-cli register-plugin acdc-blockscout
```
5. Launch blockscout
```
audius-cli launch-plugins
```
6. Restart caddy container

### CODE REVIEW

**Comment by michellebrier** [comment]
> given that this will understandably chew up resources can we add a `profile` to this so it can be stood up on some nodes only. fine for all of stage, but would like foundation only in prod.

added in https://github.com/AudiusProject/audius-docker-compose/pull/373/commits/74ea02d3c09aca42fe7b41cc838d9af7bb56dd8f

File: audius-cli [hunk 1]
```diff
@@ -272,9 +272,10 @@ def health_check(ctx, service):
 @click.option("-y", "--yes", is_flag=True)
 @click.option("--seed", is_flag=True)
 @click.option("--chain", is_flag=True)
+@click.option("-b", "--block-explorer", is_flag=True)
```
Comment (raymondjacobson) [inline audius-cli:275]: imo we should use the plugin system for this

Comment (raymondjacobson) [reply audius-cli:275]: https://github.com/AudiusProject/audius-docker-compose/blob/stage/discovery-provider/docker-compose.plugins.yml#L45

e.g. these containers will only start up if you have run
```
audius-cli register-plugin block-explorer
```

Comment (michellebrier) [reply audius-cli:275]: good call, will add

**Pull request merged**